sdk/healthbot/Azure.ResourceManager.Healthbot/src/autorest.md

```diff
@@ -0,0 +1,51 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+csharp: true
+library-name: Healthbot
+namespace: Azure.ResourceManager.Healthbot
```

Comment:
Why did we use bot instead of `Bot` in the namespace? Is it going to keep consistent with track 1?

Reply:
Yes, track 1 is lowercase